site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -65,54 +68,320 @@ LAST SEEN   TYPE      REASON              OBJECT
 5s          Warning   ResourceDeleted     nginxgateway/ngf-config                          NginxGateway configuration was deleted; using defaults
 ```
 
-##### Logs
+#### Get shell access to NGINX container
+
+Getting shell access to containers allows developers and operators to view the environment of a running container, see its logs or diagnose any problems. To get shell access to the NGINX container, use `kubectl exec`:
+
+```shell
+kubectl exec -it -n nginx-gateway  <ngf-pod-name> -c nginx /bin/sh
+```
+
+#### Logs
 
 Logs from the NGINX Gateway Fabric control plane and data plane can contain information that isn't available to status or events. These can include errors in processing or passing traffic.
 
-To see logs for the control plane container:
+1. Container Logs
+
+    To see logs for Control plane container:
+
+    ```shell
+    kubectl -n nginx-gateway logs <ngf-pod-name> -c nginx-gateway
+    ```
+
+    To see logs for Data plane container:
+
+    ```shell
+    kubectl -n nginx-gateway logs <ngf-pod-name> -c nginx
+    ```
+
+1. Error Logs
+
+    To see error logs for control plane and data plane containers:
+
+    For _nginx-gateway_ container, you can `grep` for the word `error` or change the log level to `error` by following steps in Modify log levels (Section 3). Once you have modified log levels, you can `grep` for the word `debug` to check debug logs for further investigation.
```

Comment:
Section 3? 

This part is a little confusing. If the user changes the log level to `error` as suggested, then they cannot search for `debug` log levels. Maybe we just simplify this to just describe how to grep for `error`.

Reply:
Saylor made a comment that we should write instructions to `grep` debug logs. A different section might be clearer